platform/linux-generic/pool/subsystem.c

```diff
@@ -0,0 +1,17 @@
+/* Copyright (c) 2017, ARM Limited. All rights reserved.
+ *
+ * Copyright (c) 2017, Linaro Limited
+ * All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+#include <odp_module.h>
+
+#define SUBSYSTEM_VERSION 0x00010000UL
```

Comment:
@muvarov I'm not sure why GitHub is flagging this comment as "outdated" as it relates to the current (v4) PR.